app/main/views/index.py

```diff
@@ -9,10 +9,28 @@
 from app.formatters import apply_html_class, convert_markdown_template
 from app.main import main
 from app.main.views.pricing import CURRENT_SMS_RATE
-from app.main.views.sub_navigation_dictionaries import features_nav, using_notify_nav
+from app.main.views.sub_navigation_dictionaries import (
+    best_practices_nav,
+    features_nav,
+    using_notify_nav,
+)
 from app.utils.user import user_is_logged_in
 from notifications_utils.url_safe_token import generate_token
 
+feature_best_practices_enabled = (
+    os.getenv("FEATURE_BEST_PRACTICES_ENABLED", "false").lower() == "true"
+)
+
+
+# Hook to check for guidance routes
+@main.before_request
+def check_guidance_feature():
+    if (
+        request.path.startswith("/best-practices")
+        and not feature_best_practices_enabled
```

Comment:
Given the notes above, this can be changed to the following:

```suggestion
        and not current_app.config["FEATURE_BEST_PRACTICES_ENABLED"]
```

Much simpler! 🙂